### ISSUE TITLE
BUG: parse-qp-output.py: error Wrong frame type parsed: 9

### ISSUE THREAD

**Issue by lars18th** [slhck/ffmpeg-debug-qp#10]
Hi,

When using the tool `parse-qp-output.py` with a `qp-values.txt` file from one mpeg2 video I see this error: `Wrong frame type parsed: 9`. However, with files from h264 videos all works.

Please, fix this error.
Thank you!

**Comment by slhck** [comment]
Please provide the video file or the QP values file in order to reproduce the issue.

**Comment by lars18th** [comment]
Hi @slhck ,

I fixed the problem. Please merge the PR #11.

The root cause is a lack of mpeg2 support. Now it's added. However, a side cause is the issue #8. With this PR you will see the cause of this specific error:
```
Wrong frame type parsed: 9
 Offending LINE : [mpeg2video @ 00000000005b6bc0] New frame, type:frame_type: B; pkt_size: 1839
Wrong frame type parsed: 2
 Offending LINE : [mpeg2video @ 00000000005b6bc0] New frame,frame_type: B; pkt_size: 1392
Wrong frame type parsed: 3
 Offending LINE : [mpeg2video @ 00000000005b6bc0] New frame_type: B; pkt_size: 1143
Wrong frame type parsed: 3
 Offending LINE : [mpeg2video @ 00000000005b6bc0] New frame, type: frame_type: I; pkt_size: 15123
```

So these inserted messages in the middle of the line require to be catched and writed in a new line.
Regards.

**Comment by slhck** [comment]
Thanks! 🎉 
I will check it out and merge.

**Comment by slhck** [comment]
Closed via #11, thanks!
